src/Hpp/StringSig.hs

```diff
@@ -195,6 +196,12 @@ sdropWhile f s = case sbreak (boolJust . f) s of
                    Just (_, _, s') -> s'
 {-# INLINE sdropWhile #-}
 
+stripR :: ByteString -> ByteString
```

Comment:
Turns out `BL.lines` leaves the `\r` at the end and this breaks the line splicing code. Hence, I added an extra function in the pipeline to strip of the terminating `\r` if it exists.